deploy.ts

```diff
@@ -94,7 +98,7 @@ export function init(options: DeploySlashInitOptions): void {
         return
       }
 
-      await (client as any)._process(d)
+      await (client as unknown as InteractionClientExt)._process(d)
```

Comment:
if you let `InteractionClientExt` extends `Client` then you don't need the `as unknown` 😉

Reply:
Right 😓

Reply:
Actually no, interfaces can't extend classess... I'll just make `_process` public cause it's prefixed with `_` which implies its for internal use only.

Reply:
```ts
type ClientType = typeof Client
```
This would allow you to extend the interface with Client